x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -371,28 +391,18 @@ async function main() {
   }
 
   if (argv.rbacUser) {
-    // Add role and user with response actions kibana privileges
-    const withRARole = await addRole(kbnClient, {
-      name: 'withResponseActions',
-      ...withResponseActionsRole,
-    });
-    if (withRARole) {
-      console.log(`Successfully added ${withRARole} role`);
-      await addUser(client, withResponseActionsUser);
-    } else {
-      console.log('Failed to add role, withResponseActions');
-    }
-
-    // Add role and user with no response actions kibana privileges
-    const noRARole = await addRole(kbnClient, {
-      name: 'noResponseActions',
-      ...noResponseActionsRole,
-    });
-    if (noRARole) {
-      console.log(`Successfully added ${noRARole} role`);
-      await addUser(client, noResponseActionsUser);
-    } else {
-      console.log('Failed to add role, noResponseActions');
+    // Add roles and users with response actions kibana privileges
+    for (const role of Object.keys(rolesMapping)) {
+      const withRole = await addRole(kbnClient, {
```

Comment:
`withRole` could now be renamed to `addedRole` as `addRole` returns the name of the added role. Or it could even return with a boolean indicating success.

Reply:
https://github.com/elastic/kibana/pull/144710/commits/9cf0ee6a5beda3073e2ec3635b219d09c7912aa0